application_form/enums.py

```diff
@@ -21,6 +21,7 @@ class ApartmentReservationCancellationReason(Enum):
         "reservation_agreement_canceled"  # Varaussopimus peruttu
     )
     TRANSFERRED = "transferred"  # Siirretty
+    APARTMENT_OFFERED = "apartment_already_offered"
```

Comment:
As every other enum's label matches its value in this project, it would be good if this was doing the same 🙂

Also, I was thinking about the name a bit... what do you think about `OTHER_APARTMENT_OFFERED`? I think it could describe the case more clearly.